packages/changed-elements-react/src/api/ChangedElementsLabelCache.ts

```diff
@@ -51,10 +51,17 @@ export class ChangedElementsLabelsCache extends ChangedElementDataCache {
         };
       },
     );
-    // Try to obtain the labels from the LabelsProvider
-    try {
-      const currentLabels = await this._currentLabelsProvider.getLabels(currentInstanceKeys);
-      const targetLabels = await this._targetLabelsProvider.getLabels(targetInstanceKeys);
+    const noLabel = IModelApp.localization.getLocalizedString(
+      "VersionCompare:versionCompare.noLabel")
```

Comment:
Doesn't this need to also be added to the `VersionCompare.json` file? I don't see a change for it in this PR.

Reply:
It was already there. I just leveraged an existing string